main.js

```diff
@@ -1,23 +1,23 @@
 import "./style.css";
 import { createHeaderComponent } from "./lib/headerComponent";
 import { createMainComponent } from "./lib/mainComponent";
-import /* createFooterComponent */ "./lib/footerComponent";
-/* import createCharacterCard from "./components/characterCard"; */
+import { createFooterComponent } from "./lib/footerComponent";
+import { fetchCharacters } from "./lib/fetchCharacter";
 
 async function renderApp() {
   const appElement = document.querySelector("#app");
 
   const headerComponent = createHeaderComponent();
 
-  const response = await fetch("https://rickandmortyapi.com/api/character");
-  const body = await response.json();
-  const characters = body.results;
+  const characterFetchElement = await fetchCharacters(
```

Comment:
Maybe just call it `characters`? That's what actually the function gives us back :) 
```suggestion
  const characters = await fetchCharacters(
```